src/OpenTelemetry.Instrumentation.GrpcNetClient/TracerProviderBuilderExtensions.cs

```diff
@@ -28,16 +28,21 @@ public static class TracerProviderBuilderExtensions
         /// Enables gRPClient Instrumentation.
         /// </summary>
         /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+        /// <param name="configureGrpcClientInstrumentationOptions">GrpcClient configuration options.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
         public static TracerProviderBuilder AddGrpcClientInstrumentation(
-            this TracerProviderBuilder builder)
+            this TracerProviderBuilder builder,
+            Action<GrpcClientInstrumentationOptions> configureGrpcClientInstrumentationOptions = null)
```

Comment:
The variable name seems too long.
It seems the console exporter took a different approach https://github.com/open-telemetry/opentelemetry-dotnet/blob/4915d74b1c4ddc213ec1a125cf940d686797f908/src/OpenTelemetry.Exporter.Console/ConsoleExporterHelperExtensions.cs#L31.